invenio_vocabularies/contrib/funders/datastreams.py

```diff
@@ -55,7 +55,8 @@ def apply(self, stream_entry, **kwargs):
         for label in record.get("labels", []):
             funder["title"][label["iso639"]] = label["label"]
 
-        funder["identifiers"] = []
+        """The ROR is always listed in identifiers, expected by serialization"""
+        funder["identifiers"] = [{"identifier": funder["id"], "scheme": "ror"}]
```

Comment:
note: at this point in time, the funder id is always present. Otherwise, an exception would have risen in line 48.